tests/test_converters.py

```diff
@@ -44,6 +50,9 @@ def test_md_converter(self):
         df = md_converter(directory_path=self.assets_folder)
         self.df_converter_check(df)
 
-    def test_pdf_converter(self):
+    def test_pdf_converter(self, line_paragraph=False):
```

Comment:
There's no need to add an arg `line_paragraph=False` here. You don't use it in this method

Reply:
Check!